### DISCUSSION TITLE
Joomla https problem...

### DISCUSSION THREAD

**Discussion by pietru2004** [joomla/joomla-cms#42147] [Q&A]
Would somebody be able to help how to make joomla work on `https://example.com:444`?
Joomla works when connecting normally (via local server ip), but when I connect via `https://example.com:444` it redirects me to `https://example.com` (port 443).
My problem is that router admin dashboard takes 443 port and it is not possible to change that port atm...
Also I use nginx proxy manager that successfully redirects few other proxy paths that don't end up as `https://example.com` and properly keep 444 port in the link...

**Comment by HLeithner** [comment]
Did you disabled the https redirect in Joomla and redirect it on nginx? Also does your nginx provide the https header?

**Reply by pietru2004** [reply]
I am not sure how to disable https redirect in joomla, but I know for sure that I have https redirect available on nginx, and it is setup to use 444

**Reply by pietru2004** [reply]
if https redirect is force https setting then it is set to none
![obraz](https://github.com/joomla/joomla-cms/assets/51274570/e5110646-fea5-47a6-8e0d-f2fe67745bf6)
